### ISSUE TITLE
'body' variable not passed as string literal

### ISSUE THREAD

**Issue by timfallmk** [meeDamian/github-release#10]
Minor issue, but I'm using the `body` input (along with `files`, etc.) to upload a markdown formatted changelog. After Github Actions expands the output from a previous step, `jq` rejects the input because it tries to parse the string. It would be nice if this entire body was treated as a string literal for such purposes.

Example:
```shell
    body: '# Change Log ## [0.4.2](https://github.com/ksync/ksync/tree/0.4.2) (2019-12-06) [Full Changelog](https://github.com/ksync/ksync/compare/0.4.1...0.4.2) **Closed issues:** - Images for release 0.4.0 missing on dockerhub [\#323](https://github.com/ksync/ksync/issues/323) - Ksync copy should copy with default container user [\#296](https://github.com/ksync/ksync/issues/296) **Merged pull requests:** - Switch to using a custom generated changelog [\#328](https://github.com/ksync/ksync/pull/328) ([timfallmk](https://github.com/timfallmk)) - Reformat tags to show version properly. [\#326](https://github.com/ksync/ksync/pull/326) ([timfallmk](https://github.com/timfallmk)) - Syncthing update [\#321](https://github.com/ksync/ksync/pull/321) ([timfallmk](https://github.com/timfallmk)) \* *This Change Log was automatically generated by [github_changelog_generator](https://github.com/skywinder/Github-Changelog-Generator)*'
  
    files: bin/ksync_darwin_amd64 bin/ksync_linux_amd64 bin/ksync_windows_amd64.exe bin/radar_darwin_amd64 bin/radar_linux_amd64 bin/radar_windows_amd64.exe
  
    allow_override: false
  env:
    GOROOT: /opt/hostedtoolcache/go/1.13.4/x64
/usr/bin/docker run --name af96b4d7ee21502ef4470890e9e162a481d4cc_4725ca --label af96b4 --workdir /github/workspace --rm -e GOROOT -e INPUT_TOKEN -e INPUT_GZIP -e INPUT_BODY -e INPUT_FILES -e INPUT_TAG -e INPUT_COMMITISH -e INPUT_NAME -e INPUT_DRAFT -e INPUT_PRERELEASE -e INPUT_ALLOW_OVERRIDE -e HOME -e GITHUB_REF -e GITHUB_SHA -e GITHUB_REPOSITORY -e GITHUB_ACTOR -e GITHUB_WORKFLOW -e GITHUB_HEAD_REF -e GITHUB_BASE_REF -e GITHUB_EVENT_NAME -e GITHUB_WORKSPACE -e GITHUB_ACTION -e GITHUB_EVENT_PATH -e RUNNER_OS -e RUNNER_TOOL_CACHE -e RUNNER_TEMP -e RUNNER_WORKSPACE -e ACTIONS_RUNTIME_URL -e ACTIONS_RUNTIME_TOKEN -e GITHUB_ACTIONS=true -v "/var/run/docker.sock":"/var/run/docker.sock" -v "/home/runner/work/_temp/_github_home":"/github/home" -v "/home/runner/work/_temp/_github_workflow":"/github/workflow" -v "/home/runner/work/ksync/ksync":"/github/workspace" af96b4:d7ee21502ef4470890e9e162a481d4cc
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   129  100   129    0     0   1000      0 --:--:-- --:--:-- --:--:--  1000
jq: invalid JSON text passed to --argjson
Use jq --help for help with command-line options,
or see the jq manpage, or online docs  at https://stedolan.github.io/jq

	ERR: POST to Github release has failed
{
  "message": "Invalid request.\n\nFor 'links/0/schema', nil is not an object.",
  "documentation_url": "https://developer.github.com/v3/repos/releases/#create-a-release"
}
##[error]Docker run failed with exit code 1
  Complete job
```

Relevant section of workflow yaml:
```yaml
    - name: Upload assets to release
      uses: meeDamian/github-release@v1.0.1
      with:
        token: ${{ secrets.GITHUB_TOKEN }}
        gzip: false
        body: '${{ steps.changelog.outputs.CHANGELOG }}'
        files: >
          bin/ksync_darwin_amd64
          bin/ksync_linux_amd64
          bin/ksync_windows_amd64.exe
          bin/radar_darwin_amd64
          bin/radar_linux_amd64
          bin/radar_windows_amd64.exe
```

**Comment by meeDamian** [comment]
Perhaps a step substituting new lines for literal `\n` on `INPUT_BODY` could do the trick.  Not quite certain what else can it be at the moment 🤔.

**Comment by meeDamian** [comment]
Please use https://github.com/actions/create-release and https://github.com/actions/upload-release-asset now instead

**Comment by meeDamian** [comment]
Fixed in `v2.0.0`
